tests/integration_tests/log_utils.py

```diff
@@ -0,0 +1,13 @@
+def ordered_items_in_text(to_verify: list, text: str):
```

Comment:
`log_utils.py` is better than `utils.py`, right???

But in all seriousness, I'm open to suggestions of a better place to put something like this.